platform/platform-configuration/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -64,7 +64,7 @@
                 default=""
         />
         <AD
-                description="Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value should be 2 minutes or greater, as users are warned when only 1 minute remains."
+                description="Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes."
```

Comment:
✏️ This is an interesting change as I think some users are still going to get into a weird state if they don't read this.
I'd think we'd want to validate and warn users from the Admin UI, but that might be beyond the scope of this change.

Reply:
Current Admin UI can't report configuration validation very well ATM. Currently if an Admin sets it to below 2 minutes there will be issues with other timeouts. The other option to defaulting was to throw an IllegalArgumentException. In that case the configuration would not be saved at all and the timeout will be the previous value, which I think is also weird behavior.

Reply:
I agree, it's kind of awkward. 
I've approved as I think this works for now, but maybe in the future of Admin UI/configuration this could get improved 👍

Reply:
With the current limitations of the admin ui configurations I think you would have to use the ConfigurationAdmin to look up your own configuration and then save it with the default value in order to keep the ui and this class instant in sync.

Reply:
FYI I think that would look something like 
```Bundle bundle = FrameworkUtil.getBundle(PlatformUiConfiguration.class);
    if (bundle != null) {
      BundleContext bundleContext = bundle.getBundleContext();
      if (bundleContext != null) {
        ServiceReference<ConfigurationAdmin> serviceReference =
                bundleContext.getServiceReference(ConfigurationAdmin.class);
        if (serviceReference != null) {
          ConfigurationAdmin configAdmin = bundleContext.getService(serviceReference);
          Configuration config = configAdmin.getConfiguration("ddf.platform.ui.config");
          config.getProperties().put("timeout",DEFEAULT_TIMEOUT_MINUTES);
          config.update();
        }
      }
    }

Reply:
You probably need to `bundleContext.ungetService(serviceReference)` when done too